### PR TITLE
Class homework no.9

### PR DESCRIPTION
**Pull request by fm1hai**

### CODE REVIEW

**Comment by fm1hai** [comment]
homework for lesson 9- class

**Comment by vlad-o-git** [comment]
Good job!
The second class shouldn't have a second method as initially requested ("at least 2 methods each")?

File: lesson_9/classes_homework_9.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    def can_be_placed(self):
+        if int(self.size) < 20:
+            return f"Due to the size, {self.size} inch, {self.brand} {self.type} is recomandated " \
+                   f"to be placed in " \
+                   f"small places like kitchen"
+        elif int(self.size) < 80:
+            return f"Due to the size, {self.size} inch, {self.brand} {self.type} is recomandated " \
+                   f"to be placed in " \
+                   f"medium places like rooms with 6 by 6 meters"
+        return f"Due to the size, {self.size} inch, {self.brand} {self.type} is recomandated " \
```
Comment (Liitle) [inline lesson_9/classes_homework_9.py]: try to use `""" """` for multiline strings instead of `\`

**Review by Liitle — CHANGES_REQUESTED** [review]
Nice job for updating the code after the first review! 

God work with the code. Just some code formatting for strings is a bit off, but good job!

**Review by mariuspodean — APPROVED** [review]

**Pull request merged**